### PR TITLE
Feature/1000 gen 150

### PR DESCRIPTION
**Pull request by rajatshuvro**

### CODE REVIEW

File: docs/data-sources/1000Genomes-snv-json.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   "afrAc":278,
+   "amrAc":97,
+   "easAc":278,
+   "eurAc":183,
+   "sasAc":170
+}
+```
+| Field            | Type         | Notes                     |
+|:-----------------|:------------:|:--------------------------|
+| allAf            | float        | 1000 Genomes Project allele frequency for all populations. Range: 0 - 1.0|
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes-snv-json.md]: The entire entry is about 1000 Genomes Project, so no need to list "1000 Genomes Project" for each of these fields. "Allele frequency for all populations. Range: 0 - 1.0" should suffice.

File: docs/data-sources/1000Genomes-sv-json.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+|-------------------|----------------|---------------------------------------------------------------------------------------------------------------------------------|
+| oneKg             | object array   |                                                                                                                                 |
+| chromosome        | string         | Ensembl-style chromosome names                                                                                                  |
+| begin             | integer        | 1-based position                                                                                                                |
+| end               | integer        | 1-based position                                                                                                                |
+| variantType       | string         | Any of the sequence alterations defined here.                                                                                   |
+| id                | string         | Identifier from the data source. Alternatively a VID                                                                            |
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes-sv-json.md]: Just leave the notes blank - it doesn't provide any additional information.

File: docs/data-sources/1000Genomes-sv-json.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+| allAn             | floating point | 1000 Genomes Project allele number for all populations. Non-zero integer.                                                       |
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes-sv-json.md]: Remove 1000 Genomes Project from Notes.

File: docs/data-sources/1000Genomes-sv-json.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+| allAf             | floating point | 1000 Genomes Project allele frequency for all populations. Range: 0 - 1.0                                                       |
+| afrAf             | floating point | 1000 Genomes Project allele frequency for the African super population. Range: 0 - 1.0                                          |
+| amrAf             | floating point | 1000 Genomes Project allele frequency for the Ad Mixed American super population. Range: 0 - 1.0                                |
+| eurAf             | floating point | 1000 Genomes Project allele frequency for the European super population. Range: 0 - 1.0                                         |
+| easAf             | integer        | 1000 Genomes Project allele frequency for the East Asian super population. Range: 0 - 1.0                                       |
+| sasAf             | integer        | 1000 Genomes Project allele frequency for the South Asian super population. Range: 0 - 1.0                                      |
+| reciprocalOverlap | floating point | Range: 0 - 1. E.g. 0.57 would indicate a 57% reciprocal overlap. Specified up to 5 decimal places(Not reported for Insertions). |
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes-sv-json.md]: Remove most of this. Just say Range: 0 - 1. In other data sources we have a separate description for reciprocal overlap.

File: docs/data-sources/1000Genomes.mdx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+Similarly, for all other variant types except insertion, END is far larger than BEGIN. The distribution of BEGIN and END for insertions is summarized below.
+
+**Insertion issues**
+* END = BEGIN for 6/165
+* END = BEGIN+2 for 93/165
+* END = BEGIN+3 for 11/165
+* END = BEGIN+4 for 11/165
+* END – BEGIN range from 5 to 1156 for others.
+
+#### Example
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes.mdx]: Generally I like to start with the example first before talking about how to parse it. Also Parsing and Example are generally at the same level.

File: docs/data-sources/1000Genomes.mdx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+#### Exceptions
+*We discard structural variants without END*
+```scss
+#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO    FORMAT  HG00096 HG00097 HG00099 HG00100 HG00101 HG00102 HG00103
+21      9495848 esv3646347      A       <INS:ME:LINE1>  100     PASS   AC=1543;AF=0.308107;AN=5008;CS=L1_umary;MEINFO=LINE1,5669,6005,+;NS=2504;SVLEN=336;SVTYPE=LINE1;TSD=null;DP=20015;EAS_AF=0.3125;AMR_AF=0.2911;AFR_AF=0.3026;EUR_AF=0.2922;SAS_AF=0.3395;VT=SV   GT      0|0     1|1     1|0     0|1     1|0     1|0     0|0
+```
+
+**CNVs in chrY**
+* No other types of structural variants exists in chrY
```
Comment (MichaelStromberg) [inline docs/data-sources/1000Genomes.mdx]: exist

File: docs/file-formats/nirvana-json-file-format.mdx [hunk 7]
```diff
@@ -9,6 +9,8 @@ import REVEL from '../data-sources/revel-json.md'
 import SpliceAI from '../data-sources/splice-ai-json.md'
 import MitoMapSmall from '../data-sources/mitomap-small-variants-json.md'
 import MitoMapSV from '../data-sources/mitomap-structural-variants-json.md'
+import ThousanGenSmall from '../data-sources/1000Genomes-snv-json.md'
```
Comment (MichaelStromberg) [inline docs/file-formats/nirvana-json-file-format.mdx]: ThousandGenomesSmall & ThousandGenomesSV

File: docs/file-formats/nirvana-json-file-format.mdx [hunk 8]
```diff
@@ -30,38 +32,38 @@ For each gene that was referenced in the transcripts found in the positions sect
 
 ## Header
 ```json
-{ 
-   "header":{ 
+{
```
Comment (MichaelStromberg) [inline docs/file-formats/nirvana-json-file-format.mdx]: I'm a bit confused about all the changes in this file... seems like a ton of lines were changed... Never mind, this was just a line ending change. No worries...

File: sidebars.js [hunk 9]
```diff
@@ -20,7 +20,8 @@ module.exports = {
         'data-sources/mitomap',
         'data-sources/revel',
         'data-sources/splice-ai',
-        'data-sources/primate-ai'
+        'data-sources/primate-ai',
+		'data-sources/1000Genomes'
```
Comment (MichaelStromberg) [inline sidebars.js]: Seems like you're using a different number of spaces.

**Review by MichaelStromberg — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 9 above)
Comment (rajatshuvro) [reply sidebars.js]: my editor was using a tab. Replaced them with spaces

File: docs/file-formats/nirvana-json-file-format.mdx (hunk 7 above)
Comment (rajatshuvro) [reply docs/file-formats/nirvana-json-file-format.mdx]: done

File: docs/data-sources/1000Genomes.mdx (hunk 5 above)
Comment (rajatshuvro) [reply docs/data-sources/1000Genomes.mdx]: I removed the Example tag. I started with the vcf line and described what is don.

**Review by haochenl — APPROVED** [review]

**Pull request merged**